website/content/docs/connect/registration/service-registration.mdx

```diff
@@ -81,6 +81,7 @@ registering a proxy instance.
     "destination_service_id": "redis1",
     "local_service_address": "127.0.0.1",
     "local_service_port": 9090,
+    "local_service_socket_path": "/tmp/redis.sock",
```

Comment:
@hashicorp/consul-docs team. I've listed this option because 'this shows all the options available'. However it conflicts with local_service_address, so this actual example doesn't work. I think this could generate confusion, but I don't have a great idea how to handle this.